src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -950,9 +950,7 @@ namespace cryptonote
       uint64_t rpc_connections_count;
       uint64_t white_peerlist_size;
       uint64_t grey_peerlist_size;
-      bool mainnet;
-      bool testnet;
-      bool stagenet;
+      std::string nettype;
```

Comment:
This needs bumping RPC version (and it's gratuitously backward incompatible)

Reply:
Good point. But currently the mainnet/stagenet bools of the DaemonInfo object do not get printed. Shall I add a bool stagenet, stick to std::string nettype, or leave it?

Reply:
A string type seems like a good idea, but it should preserve the booleans for code that expects them.

Reply:
ack

Reply:
Is this field used anywhere?

Reply:
This patch seems irrelevant for the regtest itself at this point, though I'm OK with adding this string field. However, it seems incomplete because there is no corresponding change in core_rpc_server.cpp.

Reply:
Adjusted